api/test/models/checkout-items/checkout-items-data-mapper-impl.test.ts

```diff
@@ -0,0 +1,136 @@
+// GLOBAL REQUIREMENTS
+import { Substitute } from '@fluffy-spoon/substitute';
+import { expect } from 'chai';
+import 'mocha';
+import { CheckoutItems, CheckoutItemsDataMapperImpl } from '../../../lib/models/checkout-items';
+import { RBAC } from '../../../lib/services/auth/RBAC/rbac';
+import { IAcl } from '../../../lib/services/auth/RBAC/rbac-types';
+import { IDataMapper } from '../../../lib/services/database';
+import { MysqlUow } from '../../../lib/services/database/svc/mysql-uow.service';
+import { Logger } from '../../../lib/services/logging/logging';
+import { IActiveHackathonDataMapper } from '../../../lib/models/hackathon/active-hackathon';
+import { ActiveHackathon } from '../../../lib/models/hackathon/active-hackathon/active-hackathon';
+import { of } from 'rxjs';
+
+function mockedQuery<T>(query, params) {
+  // @ts-ignore
+  return Promise.resolve({ query, params });
+}
+
+let checkoutItemsDataMapper: IDataMapper<CheckoutItems>;
```

Comment:
Wasn't a separate interface defined for checkoutItems?

Reply:
Should I be using the ICheckoutApiModel interface in checkout-items.ts?

Reply:
Yes